app/main.py

```diff
@@ -1,7 +1,19 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import requests
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def get_weather() -> str:
+    key = os.environ.get("KEY")
+    filtering = "Paris"
+    url = "http://api.weatherapi.com/v1/current.json?key="
+    response = requests.get(url + f"{key}&q=" + f"{filtering}&api=no")
```

Comment:
it is better to avoid using f-strings with url, u can use param arg for get method to pass ur url params